src/explore-education-statistics-common/src/modules/release/components/NationalStatisticsSection.tsx

```diff
@@ -34,17 +39,24 @@ export default function NationalStatisticsSection({
         <li>well explained and readily accessible</li>
       </ul>
       <p>
-        Once designated as National Statistics it's a statutory requirement for
-        statistics to follow and comply with the Code of Practice for Statistics
-        to be observed.
+        Our statistical practice is regulated by the Office for Statistics
+        Regulation (OSR).
+      </p>
+      <p>
+        OSR sets the standards of trustworthiness, quality and value in the{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
```

Comment:
do we want this to be a React `<Link>` component?

Reply:
As above.